trefoil/render/renderers/tests/test_renderers.py

```diff
@@ -20,7 +20,6 @@ def test_stretched_renderer(tmpdir):
     assert renderer.name == 'stretched'
 
     img = renderer.render_image(data)
-    assert len(img.getpalette()) / 3 == 256

```

Comment:
This assertion was failing, but I thought it wasn't needed if all of the subsequent assertions were passing.

Reply:
How was it failing? The value didn't equal 256? Or it was causing an exception?

Reply:
Thanks for the suggestion. The images look identical.